core/modules/core/src/com/haulmont/yarg/formatters/impl/xlsx/Document.java

```diff
@@ -278,7 +279,12 @@ public void clearWorkbook() {
                 throw new ReportFormattingException("An error occurred while clearing docx4j workbook", e);
             }
         }
-        workbook.getDefinedNames().getDefinedName().clear();
+    }
+
+    public void excludeDefinedNames(BandData rootBand) {
```

Comment:
I would name the method `clearBandDefinedNames` - in this case, it will "work in pair" with the `clearWorkbook` method. It also worth writing a method javadoc there - "Method clears defined names associated with band data and leaves all other defined names in the workbook".